src/main/java/com/drinchev/projectlabel/utils/UtilsFont.java

```diff
@@ -42,4 +43,13 @@ public static Font setAttributes(Font font, Float weight, Float size) {
         return font.deriveFont(attributes);
     }
 
+    public static JBFont getStatusBarItemFont() {
+        try {
+            return JBFont.smallOrNewUiMedium();
```

Comment:
I'm looking at the code and I also have this error. 

Was wondering what bad can happen if we just replace this with `.medium()` and call it a day? :D 

![image](https://user-images.githubusercontent.com/202919/230397397-6f2e8abf-4b6c-47f2-a611-14d8642f3686.png)

Reply:
The javadoc of `smallOrNewUiMedium()` explains why I used it:

> In new UI small() font should be used only in exceptional cases when the medium size won't fit at all. This is a temporary method for backward compatibility with old UI, which returns medium() for new UI and small() for old UI. Will be replaced by medium() after full migration to new UI

So, if we just use `medium()` we will break the UI for folks that have not migrated to the "new UI".

How about we do something with reflection?

```java
    public static JBFont getStatusBarItemFont() {
        try {
            // while the new UI is experimental, we have to carefully choose the font size
            if (JBFont.class.getDeclaredMethod("smallOrNewUiMedium") != null) {
                return Registry.is("ide.experimental.ui") ? JBFont.medium() : JBFont.small();
            }
        } catch (NoSuchMethodException e) {
            return JBFont.medium(); // the method has been removed, medium should be the new normal
        }
        throw new IllegalStateException("Unable to determine the status bar font size");
    }
```

Reply:
(Un)fortunately the plugin verification passed with `JBFont.medium()` before I was able to read your comment.

Reading [an official reply](https://intellij-support.jetbrains.com/hc/en-us/community/posts/8954459501970-How-to-determine-if-new-ui-is-enabled-), it seems that `Registry.is("ide.experimental.ui")` might break in the future. I hope there's a better way of detecting this, however if I don't find anything else I will give this a try :)